index.js

```diff
@@ -349,13 +354,15 @@ async function fastifyStatic (fastify, opts) {
             continue
           }
           routes.add(route)
-          fastify.head(route, routeOpts, function (req, reply) {
-            pumpSendToReply(req, reply, '/' + file, rootPath)
-          })
-
-          fastify.get(route, routeOpts, function (req, reply) {
-            pumpSendToReply(req, reply, '/' + file, rootPath)
-          })
+          const matchedRouteOpts = {
+            ...routeOpts,
+            method: ['HEAD', 'GET'],
+            url: route,
+            handler
```

Comment:
Should we set explicitly exposeHeadRoutes to false to avoid having globally set exposeHeadRoutes true having an effect on the Head route Registration?

Reply:
Not needed, it doesn't add for array methods.